src/demo-app/baseline/baseline-demo.html

```diff
@@ -14,9 +14,12 @@
       <input mdInput placeholder="Input" value="Text Input">
     </md-form-field>
     | Text 5 |
-    <md-select placeholder="Select">
-      <md-option value="option">Option</md-option>
-    </md-select>
+    <md-form-field>
```

Comment:
API-wise, do we really want people to have to wrap every select in a form field? E.g. if you were using option groups, you'd have to write 4 levels of HTML. Aside from having to proxy a few `@Input`-s, what are the cons of having the form field be consumed by `md-select` internally?